src/main/kotlin/uk/gov/justice/hmpps/offenderevents/service/OffenderEventStore.kt

```diff
@@ -6,37 +6,32 @@ import org.springframework.stereotype.Service
 data class StoredMessage(val eventType: EventType, val message: Message)
 
 @Service
-class OffenderEventStore(@Value("\${ui.pageSize}") val pageSize: Int) {
+class OffenderEventStore(@Value("\${model.cacheSize}") val cacheSize: Int,
+                         private val store: MutableList<StoredMessage> = mutableListOf()) : MutableList<StoredMessage> by store {
```

Comment:
Delegate to collection rather than compose collection means we can override add more easily